build/dependencies.props

```diff
@@ -6,6 +6,6 @@
     <RuntimeFrameworkVersion Condition="'$(TargetFramework)'=='netcoreapp2.0'">2.0.0-*</RuntimeFrameworkVersion>
     <TestSdkVersion>15.3.0-*</TestSdkVersion>
     <XunitVersion>2.3.0-beta2-*</XunitVersion>
-    <MoqVersion>4.7.1</MoqVersion>
+    <MoqVersion>4.7.58</MoqVersion>
```

Comment:
Seems like overkill. We're already in agreement that we want to upgrade. Ryan just hasn't gotten to this repos yet.  https://github.com/aspnet/Coherence-Signed/issues/583.